com.unity.render-pipelines.universal/Shaders/Utils/Blit.shader

```diff
@@ -16,45 +16,28 @@ Shader "Hidden/Universal Render Pipeline/Blit"
             // Required to compile gles 2.0 with standard srp library
             #pragma prefer_hlslcc gles
             #pragma exclude_renderers d3d11_9x
-            #pragma vertex Vertex
-            #pragma fragment Fragment
 
+            #pragma vertex FullscreenVert
+            #pragma fragment Fragment
             #pragma multi_compile _ _LINEAR_TO_SRGB_CONVERSION
+            #pragma multi_compile _ _USE_DRAW_PROCEDURAL
```

Comment:
Same here: adding more "multi_compile _ " will only increase shader build times. For fill-rate intensive postprocessing, use quad mesh is sufficient.